### DISCUSSION TITLE
110

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#33] [General]
# 潮流周刊第110期 - 大阪夜景

封面来源于 2018 年冬季夜晚在大阪 Abeno Harukas 高楼下俯拍大阪城的照片，很是漂亮，日本是放开后最想再去一趟的地方，整体给人感受不错。

https://weekly.tw93.fun/posts/110-%E5%A4%A7%E9%98%AA%E5%A4%9C%E6%99%AF

**Comment by kazimics** [comment]
He3 好像并不是开源的

**Reply by tw93** [reply]
哈哈，对，我的标题有误导，当做潮流工具就好喽
